### ISSUE TITLE
suggest to add 2 dependencies to fcitx5

### ISSUE THREAD

**Issue by superiums** [void-linux/void-packages#39883]
after install fcitx5 and fcitx5-rime, i found the rime input method not work in wayland. (works in xorg).

after lots of tries, i found it works after install these 2 pkgs.
`fcitx5-gtk+2`, `fcitx5-gtk+3`

so i addvice to add them to the dependencies.

**Comment by sgn** [comment]
No, Some people don't use GTK at all, I intend to remove `gtk+` supports soon.
